specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -396,6 +396,206 @@
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private endpoint."
+          },
+          {
+            "name": "privateDnsZoneGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private dns zone group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting privateDnsZoneGroup resource.",
+            "schema": {
+              "$ref": "#/definitions/PrivateDNSZoneGroup"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get private dns zone group": {
+            "$ref": "./examples/PrivateEndpointDnsZoneGroupGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "PrivateDnsZoneGroup"
+        ],
+        "operationId": "PrivateDnsZoneGroups_CreateOrUpdate",
+        "description": "Creates or updates a private dns zone group in the specified private endpoint.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private endpoint."
+          },
+          {
+            "name": "privateDnsZoneGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private dns zone group."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateDNSZoneGroup"
```

Comment:
Dns here and at other places.

Reply:
Gotcha, Pending, will push once I've resolved all of the comments